hack/test-templates.sh

```diff
@@ -360,7 +398,8 @@ if [[ -n ${CHECKS["restart"]} ]]; then
 	fi
 
 	INFO "Stopping \"$NAME\""
-	limactl stop "$NAME"
+	# TODO https://github.com/lima-vm/lima/issues/3221
+	limactl stop "$NAME" || [ "$(uname -o)" = "Msys" ]
```

Comment:
The result of `$(uname -o)` can be cached in the top of the script

Reply:
Implemented